packages/app/src/components/inspector/TextStyleInspector.tsx

```diff
@@ -0,0 +1,377 @@
+              <StretchHorizontallyIcon />
+            </RadioGroup.Item>
+            <RadioGroup.Item value="vertical" tooltip="Vertical">
+              <StretchVerticallyIcon />
+            </RadioGroup.Item>
+            <RadioGroup.Item value="outside" tooltip="Outside">
+              <SquareIcon />
+            </RadioGroup.Item>
+          </RadioGroup.Root>
+        </LabeledElementView>
+      </InspectorPrimitives.Row>
+
+      <Spacer.Vertical size={8} />
+      <InspectorPrimitives.Row>
+        <RadioGroup.Root
+          id={'text-horizontal-aligment'}
+          value={fontHorizontalAlignment}
+          onValueChange={onChangeFontHorizontalAlignment}
+        >
+          <RadioGroup.Item value="left" tooltip="Left">
+            <TextAlignLeftIcon />
+          </RadioGroup.Item>
+          <RadioGroup.Item value="center" tooltip="Center">
+            <TextAlignCenterIcon />
+          </RadioGroup.Item>
+          <RadioGroup.Item value="Right" tooltip="Right">
+            <TextAlignRightIcon />
+          </RadioGroup.Item>
+          <RadioGroup.Item value="justify" tooltip="Justify">
+            <TextAlignJustifyIcon />
+          </RadioGroup.Item>
+        </RadioGroup.Root>
+      </InspectorPrimitives.Row>
+
+      <Spacer.Vertical size={8} />
+      <InspectorPrimitives.Row>
+        <RadioGroup.Root
+          id={'text-vertical-aligment'}
+          value={fontVerticalAlignment}
+          onValueChange={onChangeFontVerticalAlignment}
+        >
+          <RadioGroup.Item value="top" tooltip="Top">
+            <PinTopIcon />
+          </RadioGroup.Item>
+          <RadioGroup.Item value="center" tooltip="Center">
+            <AlignCenterVerticallyIcon />
+          </RadioGroup.Item>
+          <RadioGroup.Item value="bottom" tooltip="Bottom">
+            <PinBottomIcon />
+          </RadioGroup.Item>
+        </RadioGroup.Root>
+      </InspectorPrimitives.Row>
+    </InspectorPrimitives.Section>
+  );
+});
+const TextOptionsRow = memo(function TextOptionsRow({
+  fontCase,
+  fontDecorator,
+  onChangeFontCase,
+  onChangeFontDecorator,
+}: TextOptionsRowProps) {
+  const id = '1';
+  const decoratorInputId = `${id}-decorator`;
+  const transformInputId = `${id}-transform`;
+  const decorator = ['None', 'Underline', 'Strikethrough'];
+
+  const renderLabel = useCallback(
+    ({ id }) => {
+      switch (id) {
+        case decoratorInputId:
+          return <Label.Label>Decoration</Label.Label>;
+        case transformInputId:
+          return <Label.Label>Transform</Label.Label>;
+        default:
+          return null;
+      }
+    },
+    [decoratorInputId, transformInputId],
+  );
+  return (
+    <InspectorPrimitives.Section>
+      <InspectorPrimitives.SectionHeader>
+        <InspectorPrimitives.Title>Text Options</InspectorPrimitives.Title>
+      </InspectorPrimitives.SectionHeader>
+      <Spacer.Vertical size={10} />
+      <InspectorPrimitives.Row id={'id'}>
```

Comment:
I'm not sure if this id is needed, but it doesn't hurt anything either. The ids are mainly for drag n drop or label stuff. If we keep it, it should be a different string, like `text-options` or something though

Reply:
I'll remove it in this case